### ISSUE TITLE
Echo middleware logs cannot be disabled.

### ISSUE THREAD

**Issue by apollo13** [wasilak/elastauth#27]
Even with a logging configuration of:
```
log_level: warn
log_format: plain
```
the echo middleware still logs all requests:
```
{"time":"2024-12-09T13:03:29.197934761Z","id":"","remote_ip":"10.7.207.20","host":"127.0.0.1:4181","method":"GET","uri":"/","user_agent":"Go-http-client/1.1","status":200,"error":"","latency":44647,"latency_human":"44.647µs","bytes_in":0,"bytes_out":0}
```

It would be nice if it could accept the configured logging format. Either way, thank you for this program :)

**Comment by wasilak** [comment]
Hey @apollo13 ! 

Thanks for reporting this. Actually, it is a great catch, something I've missed so far!

I've fixed this in 1d85da5656ddfcc1a540dc3ac0bce0c5545ef328 - now echo respects any log level and format passed to application.

It is in version [v1.1.0](https://github.com/wasilak/elastauth/releases/tag/v1.1.0), let me know if it works for you :)

**Comment by apollo13** [comment]
Yes that look way better, thank you!
